src/index.ml

```diff
@@ -686,7 +698,11 @@ struct
           > 0)
     in
     if do_merge then
```

Comment:
This variable probably wants renamed to something like `log_limit_reached`, since it's not determining whether we merge any more.